staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -161,6 +185,9 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"List of allowed origins for CORS, comma separated.  An allowed origin can be a regular "+
 		"expression to support subdomain matching. If this list is empty CORS will not be enabled.")
 
+	fs.StringSliceVar(&s.HSTSHeaders, "strict-transport-security-headers", s.HSTSHeaders, ""+
```

Comment:
please provide an example in the usage.  I may have misunderstood the code.  Are these the hsts directives that go into a single header?  The code suggests the header cannot be repeated?

Reply:
Yea, all hsts directives go into a single header